README.md

```diff
@@ -133,27 +123,31 @@ View the [features](https://surrealdb.com/features), the latest [releases](https
 - [x] Connect and query directly from web-browsers and client devices
 - [x] Use embedded JavaScript functions for custom advanced functionality
 
-<h2><img height="20" src="/img/documentation.svg">&nbsp;&nbsp;Documentation</h2>
+<h2><img height="20" src="./img/documentation.svg">&nbsp;&nbsp;Documentation</h2>
 
 For guidance on installation, development, deployment, and administration, see our [documentation](https://surrealdb.com/docs).
 
-<h2><img height="20" src="/img/installation.svg">&nbsp;&nbsp;Installation</h2>
+<h2><img height="20" src="./img/installation.svg">&nbsp;&nbsp;Installation</h2>
 
 SurrealDB is designed to be simple to install and simple to run - using just one command from your terminal. In addition to traditional installation, SurrealDB can be installed and run with HomeBrew, Docker, or using any other container orchestration tool such as Docker Compose, Docker Swarm, Rancher, or in Kubernetes.
 
-<h4><a href="https://surrealdb.com/install#gh-dark-mode-only"><img width="20" src="/img/white/apple.svg"></a><a href="https://surrealdb.com/install#gh-light-mode-only"><img width="20" src="/img/black/apple.svg"></a>
+<!-- <h4><a href="https://surrealdb.com/install#gh-dark-mode-only"><img width="20" src="/img/white/apple.svg"></a><a href="https://surrealdb.com/install#gh-light-mode-only"><img width="20" src="/img/black/apple.svg"></a>
```

Comment:
Here the Apple logo is deleted without replacement

Reply:
fixed in new commit